src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx

```diff
@@ -80,26 +80,33 @@ To deploy these resources to your cloud sandbox, run the following CLI command i
 
 <InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
-```bash
+```bash title="Terminal" showLineNumbers={false}
 npx amplify sandbox
 ```
 
 </InlineFilter>
 
 <InlineFilter filters={["android"]}>
 
-```bash
-npx amplify sandbox --config-format json-mobile --config-out-dir <path_to_src/main/res/raw_folder>
+```bash title="Terminal" showLineNumbers={false}
+npx amplify sandbox --config-format json-mobile --config-out-dir <path_to_android_project>/src/main/res/raw/
```

Comment:
Aren't we running this in the base path of the project? Then we can run the following. 

Also IIRC, if we don't have the folder already, it does not create it so having a call out to mention users need to create the "raw" folder (like we do in mobile support) is important. 

```suggestion
npx amplify sandbox --config-format json-mobile --config-out-dir app/src/main/res/raw/
```

Reply:
Now it creates the raw folder when provided in the path.